src/main/java/jenkins/scm/impl/subversion/SubversionSCMSource.java

```diff
@@ -231,8 +231,7 @@ public synchronized String getUuid() {
     protected void retrieve(@CheckForNull SCMSourceCriteria criteria,
                             @NonNull final SCMHeadObserver observer,
                             @CheckForNull SCMHeadEvent<?> event,
-                            @NonNull TaskListener listener)
-            throws IOException {
+                            @NonNull TaskListener listener) throws IOException, InterruptedException {
```

Comment:
Do we still need to add `InterruptedException` or was that just required for a dependency update?